pkg/configurator/client.go

```diff
@@ -138,9 +141,9 @@ func (c *Client) getConfigMap() *osmConfig {
 		UseHTTPSIngress:             getBoolValueForKey(configMap, useHTTPSIngressKey),
 
 		ZipkinTracing:  getBoolValueForKey(configMap, zipkinTracingKey),
-		ZipkinAddress:  getStringValueForKey(configMap, zipkinAddressKey),
-		ZipkinPort:     getIntValueForKey(configMap, zipkinPortKey),
-		ZipkinEndpoint: getStringValueForKey(configMap, zipkinEndpointKey),
```

Comment:
Can these helpers be modified to log a message and return default values for `string`, `int` types? 
Would that avoid the need for separate helpers considering the defaults are nothing but the zero values for those types?

Reply:
I have removed the helpers and replaced them with the previous methods. PTAL